project/plugins.sbt

```diff
@@ -1,3 +1,4 @@
 resolvers += Resolver.sonatypeRepo("releases")
 
-addSbtPlugin("io.frees" % "sbt-freestyle" % "0.3.6")
\ No newline at end of file
+addSbtPlugin("io.frees" % "sbt-freestyle" % "0.3.7")
+addSbtPlugin("io.frees" % "sbt-frees-protogen" % "0.0.13-SNAPSHOT")
```

Comment:
`sbt-frees-protogen`:`0.0.13` should have been released/published.

Reply:
Sorry, I forgot to change that. I am going to change and push it. Thanks for the review.